Change in UI

raviikushh opened the pull request:
- Made Changes in UI
- Updated Navbar and Footer , layout also
- increased font size of Question

raviikushh commented:
Can You Please Check if everything is alright or not before merging

eduardconstantin commented:
This is a bit too much.
The blue border doesn't look too good.
If you can just replace the image for the question with just a text that would be great.

raviikushh commented:
> This is a bit too much.
> The blue border doesn't look too good.
> If you can just replace the image for the question with just a text that would be great.
> 
> This is how it looks on my end
> ![image](https://github.com/eduardconstantin/azure-fundamentals/assets/9199763/2c034c41-b708-4f79-b6b1-54710efd570b)
> 

Okay I'll fix it.

raviikushh commented:
Made few changes
- Updated UI and removed cyan color
- Adjusted Question Size so that it is clearly visible to user with good font 
- Also added some hover effect in the options and selection
- Changed the color of selection of option
- The biggest challenge was the question but I tried my best to adjust it according to user so that it looks good. It's very difficult to convert it to text because lots of files are there which are using to convert it to image and few things I'm unable to understand. We could convert it to text by using Tesseract Js bt it's too complicated here.
- I request you to review it and check whether it's looking good and accept my PR

eduardconstantin commented on components/Footer.tsx:
```diff
@@ -1,7 +1,10 @@
   return (
-    <footer className="flex justify-center items-baseline py-3">
-      <p className="text-white text-sm">Built with 💻 using NextJS, TailwindCSS and CosmosDB</p>
+    <footer className="flex justify-center items-baseline py-3 border-t-[1px] border-gray-300 mt-6">
```
we don't need a border for footer
```suggestion
    <footer className="flex justify-center items-baseline py-3">
```

eduardconstantin commented on components/QuizForm.tsx:
```diff
@@ -53,7 +55,7 @@ const QuizForm: React.FC<Props> = ({
     <form onSubmit={handleSubmit(onSubmit)}>
       <div className="relative min-h-40">
-        <div className="flex justify-center">
+        <div className="flex justify-center mt-[-10px] mb-[-30px]">
```
don't add negative margins
```suggestion
        <div className="flex justify-center">
```

eduardconstantin commented on components/TopNav.tsx:
```diff
@@ -11,19 +11,18 @@ const TopNav = () => {
       </div>
       <div className="flex items-center flex-col w-full">
-        <p className="font-bold text-4xl leading-7">AZURE</p>
+        <p className="font-bold text-4xl leading-7 cursor-pointer">AZURE</p>
```
no need to change the cursor since it's not a link
```suggestion
        <p className="font-bold text-4xl leading-7">AZURE</p>
```

eduardconstantin commented on pages/api/og.tsx:
```diff
@@ -13,14 +13,20 @@ export default function handler(req: NextRequest) {
       (
-        <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center">
-          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-2xl"} font-sans text-stone-300`}>{question}</p>
+        <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center mt-20 ">
```
```suggestion
        <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center ">
```

eduardconstantin commented on pages/api/og.tsx:
```diff
@@ -13,14 +13,20 @@ export default function handler(req: NextRequest) {
+        <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center mt-20 ">
+          <p
+            tw={`px-8 ${
+              width < 640 ? "text-5xl" : "text-3xl"
```
```suggestion
              width < 640 ? "text-3xl" : "text-2xl"
```

eduardconstantin commented on pages/index.tsx:
```diff
@@ -4,17 +4,19 @@ import ExamLink from "@azure-fundamentals/components/ExamLink";
 const Home: NextPage = () => {
   return (
-    <div className="mx-auto w-5/6 sm:w-2/3 lg:w-2/3 xl:w-2/4 text-center">
+    <div className="mx-auto w-5/6 sm:w-2/3 lg:w-2/3 xl:w-2/4 text-center border-[1px] border-slate-500 mt-20 rounded-lg">
```
```suggestion
    <div className="mx-auto w-5/6 sm:w-2/3 lg:w-2/3 xl:w-2/4 text-center">
```

eduardconstantin commented on pages/index.tsx:
```diff
@@ -4,17 +4,19 @@ import ExamLink from "@azure-fundamentals/components/ExamLink";
+        and master over 480 questions at your own pace with our practice mode.
       </p>
-      <div className="flex max-sm:flex-col max-sm:align-center justify-center gap-4">
+      <div className="flex max-sm:flex-col max-sm:align-center justify-center gap-10 m-16">
```
```suggestion
      <div className="flex max-sm:flex-col max-sm:align-center justify-center gap-10 sm:mx-16">
```

eduardconstantin commented on pages/practice/index.tsx:
```diff
@@ -48,10 +52,14 @@ const Practice: NextPage = () => {
 
   return (
-    <div className="py-10 px-5 sm:p-10 mx-auto w-5/6 sm:w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
+    <div className="py-10 px-5 sm:p-10 mx-auto w-5/6 sm:w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg mt-20">
```
```suggestion
    <div className="py-10 px-5 sm:p-10 mx-auto w-5/6 sm:w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
```

raviikushh commented:
updated all requested changes

eduardconstantin reviewed (approved):
Great! Thank you for your contribution!

raviikushh commented:
You're most welcome., means a lot.
